src/components/talks.jsx

```diff
@@ -71,29 +99,36 @@ export default React.createClass({
     </Row>;
   },
 
-  render() {
-    return <div className="seminars">
-      <div className="blue-section-wrapper">
-        <Grid className="blue-section">
-          <Row>
-            <Col xs={12}>
-              <Header isLoggedIn={this.props.isLoggedIn}
-                      login={this.props.login}
-                      logout={this.props.logout} />
-            </Col>
-          </Row>
-          <Row>
-            <Col xs={12}>
-              <h1>Seminarii InfoEducație</h1>
-              <h2>Ediția {this.props.current.edition.year}</h2>
-            </Col>
-          </Row>
-          <Row className="big-spacing" />
-        </Grid>
-      </div>
-      <Grid>
-        {this.state.talks.map(this.renderSeminar)}
-      </Grid>
-   </div>;
+  renderAuthorImage(author, index) {
+    return <li className="author-image" key={index}>
+      <img src={this.getGravatarImage(author.email_md5)} />
+    </li>;
+  },
+
+  renderAuthorText(author, index) {
+    return <li key={index} className="author-text">
+      <h5 className="author-name">{author.name}</h5>
+      <h6 className="author-job">{author.job}</h6>
+    </li>;
+  },
+
+  onEditionChange(edition) {
+    this.getTalks(edition.id);
+    this.setState({ selectedEditionYear: edition.year });
+  },
+
+  getTalks(editionId=undefined) {
+    let data = {};
+    if (editionId) {
+      data.edition = editionId;
+    }
+
+    ajax("talks.json", (data) => {
```

Comment:
Explicit is better than implicit. Cred ca daca o faci un pic mai verbose, ar fi mai readable.